lib/util/copyToClipboard_utils.dart

```diff
@@ -0,0 +1,17 @@
+
+
+import 'package:flutter/services.dart';
+
+class CopyToClipboard{
```

Comment:
No need for create a class for really a 1-liner. Delete and use it inline please.

Reply:
okay i fixed